### PR TITLE
Create connection on project page

### PR DESCRIPTION
**Pull request by IstvanM**
Tasks:

- [x] Tests updated
- [x] UI issues fixes
image
image
image

### CODE REVIEW

File: mitzu/webapp/pages/manage_connection.py [hunk 1]
```diff
@@ -14,19 +14,75 @@
...
+
+CONPAGE_INDEX_TYPE = "connections_page"
+CONPAGE_TEST_LOADING_ID = "connections_page_test_loading"
+
+CONPAGE_CC_CONFIG = MCC.ConnectionComponentConfig(
+    index_type=CONPAGE_INDEX_TYPE, test_loading_id=CONPAGE_TEST_LOADING_ID
+)
+
+
+def create_confirm_dialog():
```
Comment (IstvanM) [inline mitzu/webapp/pages/manage_connection.py:45]: Just moved from the mamnage_connection_components.py

File: mitzu/webapp/pages/manage_connection.py [hunk 2]
```diff
@@ -151,3 +252,61 @@ def save_button_clicked(
...
+
+
+@callback(
+    Output(CONFIRM_DIALOG_ACCEPT, "n_clicks"),
+    Input(CONFIRM_DIALOG_ACCEPT, "n_clicks"),
+    State(MITZU_LOCATION, "pathname"),
+    prevent_initial_call=True,
+)
+@restricted
+def delete_confirmed_clicked(n_clicks: int, pathname: str) -> int:
```
Comment (IstvanM) [inline mitzu/webapp/pages/manage_connection.py:264]: Just moved from the mamnage_connection_components.py

File: mitzu/webapp/pages/projects/manage_project_component.py [hunk 3]
```diff
@@ -28,6 +32,67 @@
...
+
+
+def create_edit_connection_modal():
+    return dbc.Modal(
+        id=EDIT_CONNECTION_MODAL,
+        children=[
+            dbc.ModalHeader(dbc.ModalTitle("Establish connection"), close_button=False),
+            dbc.ModalBody(
+                children=[
+                    MCC.create_manage_connection_component(
```
Comment (IstvanM) [inline mitzu/webapp/pages/projects/manage_project_component.py:66]: Reusing the manage_connection_components. As that only holds complex index components.

File: mitzu/webapp/pages/projects/manage_project_component.py [hunk 4]
```diff
@@ -164,12 +303,22 @@ def create_discovery_settings(project: Optional[M.Project]) -> bc.Component:
...
-                    The discovery step scrapes the data warehouse tables for the columns and it's values.""",
+                    "Discovery settings",
+                    html.I(
+                        className="bi bi-question-circle ms-1",
+                        id="discovery_settings_help",
+                    ),
                 ],
                 className="mb-3 lead",
             ),
+            dbc.Popover(
```
Comment (IstvanM) [inline mitzu/webapp/pages/projects/manage_project_component.py:314]: Help popovers.
image

File: mitzu/webapp/pages/projects/manage_project_component.py [hunk 5]
```diff
@@ -254,3 +414,127 @@ def create_explore_settings(project: Optional[M.Project]) -> bc.Component:
...
+    Output({"type": PROJECT_INDEX_TYPE, "index": PROP_CONNECTION}, "data"),
+    Output(SAVE_CONNECTIONS_INFO, "children"),
+    Input(CLOSE_CONNECTION_BUTTON, "n_clicks"),
+    Input(SAVE_CONNECTION_BUTTON, "n_clicks"),
+    Input(NEW_CONNECTION_BUTTON, "n_clicks"),
+    State({"type": EDIT_PAGE_CON_INDEX_TYPE, "index": ALL}, "value"),
+    prevent_initial_call=True,
+)
+@restricted
+def modal_open_handler(close: int, save: int, new: int, inputs: Dict):
```
Comment (IstvanM) [inline mitzu/webapp/pages/projects/manage_project_component.py:458]: This can save the connection in the projects page

**Review by medgyes — COMMENTED** [review]
I've found an issue: only one connection can be added through the projects page. The second time I open the modal it's already filled out and after I change something it's not added as a new one.

**Comment by IstvanM** [comment]
thx, will fix

**Pull request merged**